### PR TITLE
The shooter is pretty developed right? right?

### PR DESCRIPTION
**Pull request by randomtestfive**
also carousel

### CODE REVIEW

File: src/org/redalert1741/steamworks/Robot.java [hunk 1]
```diff
@@ -146,6 +156,7 @@ public void autonomousPeriodic()
 	@Override
     public void teleopInit()
     { setupPeriodic("teleop")
+	; collect = false;
```
Comment (snydecor000) [inline src/org/redalert1741/steamworks/Robot.java:159]: Great!

File: src/org/redalert1741/steamworks/Robot.java [hunk 2]
```diff
@@ -211,13 +222,36 @@ else if(driver.getBumper(Hand.kRight))
...
     		manip.setInput(0, 0);
     	}
     	
+    	if(driver.getAButton())
+    	{
+    		shooter.setSpeed(-2600);
+    	}
+    	else
+    	{
+    		shooter.setSpeed(0);
```
Comment (snydecor000) [inline src/org/redalert1741/steamworks/Robot.java:238]: cool

File: src/org/redalert1741/steamworks/SwerveModule.java [hunk 3]
```diff
@@ -44,7 +44,7 @@ public SwerveModule(CANTalon d, CANTalon a, AnalogInput e, String name)
 		drive = d;
 		drive.setPID(SpeedP, SpeedI, SpeedD);
 		drive.setFeedbackDevice(FeedbackDevice.QuadEncoder);
-		drive.reverseSensor(true);
+		drive.reverseSensor(false);
```
Comment (snydecor000) [inline src/org/redalert1741/steamworks/SwerveModule.java:47]: whoops, yea, thanks for fixing that

**Review by snydecor000 — APPROVED** [review]
Crikey M8, thats a Gr8 merge, I R8 8/8

File: src/org/redalert1741/steamworks/Shooter.java [hunk 4]
```diff
@@ -35,6 +35,14 @@ public Shooter(CANTalon m)
...
 	public void setSpeed(double rpm)
 	{
+		if(rpm == 0)
+		{
+			flyWheel.changeControlMode(TalonControlMode.PercentVbus);
+		}
+		else
+		{
+			flyWheel.changeControlMode(TalonControlMode.Speed);
+		}
```
Comment (dracco1993) [inline src/org/redalert1741/steamworks/Shooter.java:45]: Just curious as to why it's like this?  Is there an issue with setting it to `0` in speed control mode?

Comment (snydecor000) [reply src/org/redalert1741/steamworks/Shooter.java:45]: It applies full negative amperage to stop it and it trips the breaker

File: src/org/redalert1741/steamworks/Robot.java [hunk 5]
```diff
@@ -230,7 +231,7 @@ else if(driver.getBumper(Hand.kRight))
     	
     	if(driver.getAButton())
     	{
-    		shooter.setSpeed(-4000);
+    		shooter.setSpeed(-2600);
```
Comment (dracco1993) [inline src/org/redalert1741/steamworks/Robot.java]: We should probably put this in separate variable... _eventually_

**Pull request merged**